ext/doc/Cassandra/Cluster/Builder.php

```diff
@@ -23,138 +23,165 @@
  *
  * @see \Cassandra::cluster()
  */
-final class Builder
-{
+final class Builder {
+
     /**
      * Returns a Cluster Instance.
-     *
      * @return \Cassandra\Cluster Cluster instance
      */
-    public function build() {}
+    public function build() { }
 
     /**
      * Configures default consistency for all requests.
      *
      * @param int $consistency A consistency level, must be one of Cassandra::CONSISTENCY_* values
-     *
      * @return Builder self
      */
-    public function withDefaultConsistency($consistency) {}
+    public function withDefaultConsistency($consistency) { }
 
     /**
      * Configures default page size for all results.
      * Set to `null` to disable paging altogether.
      *
      * @param int|null $pageSize default page size
-     *
      * @return Builder self
      */
-    public function withDefaultPageSize($pageSize) {}
+    public function withDefaultPageSize($pageSize) { }
 
     /**
      * Configures default timeout for future resolution in blocking operations
      * Set to null to disable (default).
      *
      * @param float|null $timeout Timeout value in seconds, can be fractional
-     *
      * @return Builder self
      */
-    public function withDefaultTimeout($timeout) {}
+    public function withDefaultTimeout($timeout) { }
 
     /**
      * Configures the initial endpoints. Note that the driver will
      * automatically discover and connect to the rest of the cluster.
      *
-     * @param string $host,... one or more ip addresses or hostnames
-     *
+     * @param string $host ,... one or more ip addresses or hostnames
```

Comment:
Agreed, we've never handled our flexible/vararg arguments well in the API docs. I'll need to think on this.